source/unified-test-format/tests/valid-pass/poc-queryable-encryption.yml

```
@@ -0,0 +1,84 @@
+description: poc-queryable-encryption
+
+schemaVersion: "1.23"
+
+runOnRequirements:
+  - minServerVersion: "8.0"
+
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents:
+        - commandStartedEvent
+        - commandSucceededEvent
+      autoEncryptOpts:
+        kmsProviders:
+          local:
+            key: Mng0NCt4ZHVUYUJCa1kxNkVyNUR1QURhZ2h2UzR2d2RrZzh0cFBwM3R6NmdWMDFBMUN3YkQ5aXRRMkhGRGdQV09wOGVNYUMxT2k3NjZKelhaQmRCZGJkTXVyZG9uSjFk
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name poc-queryable-encryption
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name coll0
+  - database:
+      id: &database1 database1
+      client: *client0
+      databaseName: &database1Name keyvault
+  - collection:
+      id: &collection1 collection1
+      database: *database0
+      collectionName: &collection1Name datakeys
+
+
+initialData:
+  - databaseName: *database1Name
+    collectionName: *collection1Name
+    documents:
+      - _id: { $binary: { base64: EjRWeBI0mHYSNBI0VniQEg==, subType: "04" } }
```

```suggestion
      - _id: &keyid { $binary: { base64: EjRWeBI0mHYSNBI0VniQEg==, subType: "04" } }
```

Suggest adding YAML anchor to reference below.